alerting/src/main/resources/org/opensearch/alerting/alerts/alert_mapping.json

```diff
@@ -83,6 +83,22 @@
         }
       }
     },
+    "finding_ids": {
+      "type" : "text",
+      "fields" : {
+        "keyword" : {
+          "type" : "keyword"
+        }
+      }
+    },
+    "related_doc_ids": {
+      "type" : "text",
+      "fields" : {
+        "keyword" : {
+          "type" : "keyword"
+        }
+      }
+    },
```

Comment:
Same document can be part of multiple findings and vice-versa. So wondering if we should we have a nested structure here?

Reply:
Lets leave this for now and modify it later in a future PR as with alias index, we may want to pass in the index info with each doc id